funnel/models/event.py

```diff
@@ -168,13 +222,33 @@ def checkin_list(cls, event):
         """
         Returns participant details along with their associated ticket types as a comma-separated string.
         """
-        participant_list = db.session.query('id', 'fullname', 'email', 'company', 'twitter', 'puk', 'key', 'checked_in', 'badge_printed', 'ticket_type_titles').from_statement(db.text('''
+        participant_list = (
+            db.session.query(
+                'id',
+                'fullname',
+                'email',
+                'company',
+                'twitter',
+                'puk',
+                'key',
+                'checked_in',
+                'badge_printed',
+                'ticket_type_titles',
+            )
+            .from_statement(
+                db.text(
+                    '''
             SELECT distinct(participant.id), participant.fullname, participant.email, participant.company, participant.twitter, participant.puk, participant.key, attendee.checked_in, participant.badge_printed,
             (SELECT string_agg(title, ',') FROM sync_ticket INNER JOIN ticket_type ON sync_ticket.ticket_type_id = ticket_type.id where sync_ticket.participant_id = participant.id) AS ticket_type_titles
             FROM participant INNER JOIN attendee ON participant.id = attendee.participant_id LEFT OUTER JOIN sync_ticket ON participant.id = sync_ticket.participant_id
             WHERE attendee.event_id = :event_id
             ORDER BY participant.fullname
-        ''')).params(event_id=event.id).all()
+        '''
+                )
+            )
+            .params(event_id=event.id)
+            .all()
+        )
```

Comment:
This part seems a bit messed up. Should the closing `'''` be indented a bit?

Reply:
Fixed